### PR TITLE
Merge rel/1.0.0-preview2 into rel/1.0.0

### PR DESCRIPTION
**Pull request by eerhardt**
Merging the fixes from preview2 into rel/1.0.0.

@livarcocc @Sridhar-MS 

/cc @piotrpMSFT @brthor

### CODE REVIEW

**Comment by brthor** [comment]
These mode changes don't look good:
 `packaging/deb/postinst 100644 → 100755`

File: test/dotnet.Tests/project.json [hunk 1]
```diff
@@ -43,6 +43,7 @@
     ]
   },
   "buildOptions": {
+    "keyFile": "../../tools/Key.snk",
```
Comment (brthor) [inline test/dotnet.Tests/project.json]: shouldn't this use the test-key?

Comment (eerhardt) [inline test/dotnet.Tests/project.json]: The other projects I checked didn't:

https://github.com/dotnet/cli/blob/rel/1.0.0/test/dotnet-compile.UnitTests/project.json#L4
https://github.com/dotnet/cli/blob/rel/1.0.0/test/Microsoft.DotNet.Configurer.UnitTests/project.json#L4

**Comment by eerhardt** [comment]
> These mode changes don't look good:
packaging/deb/postinst 100644 → 100755

Why does this keep happening?  Thanks, `git merge`.

**Comment by Sridhar-MS** [comment]
Installer changes LGTM

**Comment by eerhardt** [comment]
test OSX x64 Release Build

File: test/dotnet.Tests/project.json (hunk 1 above)
Comment (brthor) [inline test/dotnet.Tests/project.json]: Seems to be ad hoc.

https://github.com/dotnet/cli/blob/rel/1.0.0/test/Microsoft.DotNet.ProjectModel.Tests/project.json#L4

**Comment by brthor** [comment]
:shipit:

**Pull request merged**